src/main/java/lotto/domain/LottoTicketGenerator.java

```diff
@@ -0,0 +1,22 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoTicketGenerator {
+    private static final List<LottoNumber> lottoNumbersCache;
```

Comment:
상수이므로 대문자로 표기되면 좋을것 같습니다 😄 
```suggestion
    private static final List<LottoNumber> LOTTO_NUMBERS_CACHE;
```